core/controllers/base.py

```diff
@@ -53,7 +53,11 @@
 
 
 def _clear_login_cookies(response_headers):
-    """Clears login cookies from the given response headers."""
+    """Clears login cookies from the given response headers.
+
+    Args:
+        response_headers: Response Header. The given response headers.
```

Comment:
Pretty sure the typeinfo for this (and some other methods) is not correct.

You will need to investigate further to determine the type of the argument (which can be determined by finding a way to execute the function and adding a logging statement of the form `print type(arg_you_want_to_find_the_type_for)`). To find a way to execute the function, you will need to trace through the codebase to understand where the function is called from, until you can find a way of triggering such a call on your local dev server by interacting as a user with your local instance of the app.

Reply:
Done